release_notes/topics/18_0_0.adoc

```diff
@@ -1,5 +1,22 @@
 = Highlights
 
+== New Operator tech-preview
+
+With this release, we're introducing a brand new {project_operator} as a tech-preview. Apart from being rewritten from
+scratch, the main user-facing change from the legacy Operator is the used {project_name} distribution – the new Operator
+uses the Quarkus distribution of {project_name}. With that, there are a few breaking changes. For details, incl.
```

Comment:
Since we changed the API group "breaking changes" is not technically correct, what do you think about something along those lines?
```suggestion
uses the Quarkus distribution of {project_name}. With that, the API (in form of Custom Resource Definitions) has changed. For details, incl.
```